frontend/device/kobo/device.lua

```diff
@@ -106,6 +106,7 @@ local KoboDragon = Kobo:new{
     hasFrontlight = yes,
     hasMultitouch = no,
     display_dpi = 265,
+    canHWInvert = no,
```

Comment:
I'll wait for @NiLuJe (who may not reply for another week or so) but I'd add a comment referring to this PR or the issue saying that it's just for after standby, otherwise that'd be confusing. (In principle you can read these as a feature list.)